data/versions.yml

```diff
@@ -26,7 +26,7 @@ riak:
 riakcscontrol:
   - ['1.0.0', '1.0.1', '1.0.2']
 dataplatform:
-  - ['0.8.0']
+  - ['0.8.0', '1.0.0']
```

Comment:
I would just remove the 0.8.0 from this. We never actually pushed that version to S3, so we don't want it to show up on the version picker.
Also, we should bump the `dataplatform: '0.8.0'` in the below `currents:` map.